apps/api.planx.uk/modules/send/email/service.ts

```diff
@@ -35,6 +36,57 @@ export async function getTeamEmailSettings(localAuthority: string) {
   return response?.teams[0];
 }
 
+interface GetFlowId {
+  lowcal_sessions: {
+    flow_id: UUID;
+  }[];
+}
+
+export async function getFlowId(sessionId: UUID) {
+  const response = await $api.client.request<GetFlowId>(
+    gql`
+      query GetFlowId($session_id: uuid!) {
+        lowcal_sessions(where: { id: { _eq: $session_id } }) {
+          flow_id
+        }
+      }
+    `,
+    {
+      session_id: sessionId,
+    },
+  );
+  return response?.lowcal_sessions[0].flow_id;
+}
```

Comment:
We should always keep out internal types in camelCase, and we can [destructure](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring) (convert) as part of the GQL query.

```suggestion
interface GetFlowId {
  lowcalSessions: {
    flowId: UUID;
  }[];
}

export async function getFlowId(sessionId: UUID) {
  const response = await $api.client.request<GetFlowId>(
    gql`
      query GetFlowId($session_id: uuid!) {
        lowcalSessions: lowcal_sessions(where: { id: { _eq: $session_id } }) {
          flowId: flow_id
        }
      }
    `,
    {
      session_id: sessionId,
    },
  );
  return response?.lowcalSessions[0].flowId;
}
```